common/components/BalanceSidebar/EquivalentValues.tsx

```diff
@@ -63,23 +68,19 @@ export default class EquivalentValues extends React.Component<Props, CmpState> {
               {key}:
             </span>{' '}
             <span className="EquivalentValues-values-currency-value">
-              {balance.isPending ? (
-                <Spinner />
-              ) : (
-                <UnitDisplay
-                  unit={'ether'}
-                  value={balance ? balance.muln(rates[currency][key]) : null}
-                  displayShortBalance={2}
-                />
-              )}
+              <UnitDisplay
+                unit={'ether'}
+                value={values[key]}
+                displayShortBalance={3}
```

Comment:
Should we up this to `5`? Low-value accounts (~$1) currently get rounded out to `0 BTC` in the equiv rates display.

Reply:
I was talking to henry about this, I think we need to have UnitDisplay show `< 0.001` when the value is below the threshold, but above 0.